src/app/core/services/filters.service.ts

```diff
@@ -145,6 +145,7 @@ export class FiltersService {
   }
 
   clearFilters(): void {
+    this.updateFilters(this.defaultFilter);
```

Comment:
What's the reasoning for adding this line? Isn't this already covered under the call to `updatePresetView`?

Reply:
After the most recent change to filters service, the preset view does not contain all filters anymore. Specifically in the current implementation, it excludes hiddenLabels. This ensures that the hidden labels filter and any future filter that is not bound to preset views are properly reset.